webpack.config.js

```diff
@@ -5,11 +5,29 @@ const htmlPlugin = new HtmlWebPackPlugin({
     filename: "./index.html"
 });
 
+const OptimizeJsPlugin = require("optimize-js-plugin");
+
+var environment = process.env.NODE_ENV;
+
+var plugins = [htmlPlugin];
+
+if (environment === "production") {
+    plugins.push(new OptimizeJsPlugin({
+        sourceMap: false
+    }));
+}
```

Comment:
it's better to rely on webpack's `mode` rather than environment variable

Reply:
ok, I found how to use it via (env, argv) => for webpack export

btw, there are some places (even official webpack one [https://webpack.js.org/concepts/mode]?) which said this `mode` might be read from `process.env`, but it doesn't work seems so - probably not actual info into webpack documentation